05_syntax_lab/02.py

```diff
@@ -1,6 +1,23 @@
-"""
-Write a Python program that randomizes 7 numbers
-and prints their sum total.
-If the sum is divisable by 7, also print the word "Boom"
-"""
-
+"""
+Syntax excersize #2
+"""
+sum = 0
+number = 0
+Boom = ""
+rand = 0
+
+from random import randint
+
+while number < 7:

```

Comment:
A for loop provides a more readable way to perform a block N times (with number of times known in advance). Please modify to use `for` loop